USAGE.md

```diff
@@ -100,7 +100,7 @@ watcher.destroy();
 
 There are four different event types passed to the watcher callback:
 
-* `exposed` - When at least one pixel of the DOM element is in the viewport
+* `exposed` - When at least one pixel of the DOM element is in the viewport, or when the DOM element has zero area but in the viewport.
```

Comment:
I think this might be a clearer way to explain the behavior:
> When any part of the DOM element is in the viewport, even if the element has zero height or width.